packages/js/src/core/user-info.ts

```diff
@@ -0,0 +1,17 @@
+import { createRequester } from '../utils/requester';
+
+export type UserInfoResponse = {
+  sub: string;
+};
+
+export const fetchUserInfo = async (
+  userInfoEndpoint: string,
+  accessToken: string,
+  fetchFunction?: typeof fetch
+): Promise<UserInfoResponse> => {
```

Comment:
I'm wondering if we should define all core function params as object pattern, instead of directly pass in the params, since we already have that pattern in other core functions (e.g. sign-in, sign-out).

Reply:
Prefer to not define type `XxxParameters` until needed

Reply:
Based on the discussion on Slack, just use simple parameters.